app/models/notification.rb

```diff
@@ -1,9 +1,15 @@
 class Notification < ApplicationRecord
   belongs_to :user
 
+  @@NOTIFICATION_TYPES = {LIKES: "likes",COMMENTS:"comments",FOLLOW:"follow",UNFOLLOW:"unfollow",ACCEPTANCE:"follow_acceptance"}
+
+  # getter
+  def self.NOTIFICATION_TYPES
+    @@NOTIFICATION_TYPES
+  end
```

Comment:
instead of this you can just do this.
```
 TYPES = {LIKES: "likes",COMMENTS:"comments",FOLLOW:"follow",UNFOLLOW:"unfollow",ACCEPTANCE:"follow_acceptance"}
```
And use it like this
```
Notification::TYPES[:LIKES]
```

Reply:
Done